qutebrowser/mainwindow/tabwidget.py

```diff
@@ -361,7 +361,7 @@ def tabSizeHint(self, index):
             A QSize.
         """
         minimum_size = self.minimumTabSizeHint(index)
-        height = self.fontMetrics().height()
+        height = config.get('tabs', 'tabbar-size')
```

Comment:
I think the setting should still have a value like `auto` by default, which then here still uses the font height - otherwise it'll be confusing for people when they change the font size and the font is just cut off (or worse - they already have a bigger font size, and suddenly the bar is smaller).

If you need help with figuring out how to add a new config type for that (i.e. `auto` or an integer >= 8), please let me know and I'll help!